sstables/sstable_directory.hh

```diff
@@ -60,6 +60,10 @@ public:
     future<semaphore_units<>> get_units(size_t count) noexcept {
         return seastar::get_units(_sem, count);
     }
+
+    future<semaphore_units<>> get_units(size_t count, abort_source& abort) noexcept {
```

Comment:
nit: Should be in previous patch that exposed that non-abortable overload

Reply:
okay (although it is not used there, but I agree it conceptually belongs there)

Reply:
Done in v8 ([63026c2](https://github.com/scylladb/scylladb/commit/63026c2151ada0845db2c8201b3367be799a130b))